client/client.go

```diff
@@ -79,9 +87,14 @@ func runSyncMenu() {
 	case 1:
 		pushFolderToServer(conn, folderPath)
 	case 2:
-		pushFileToServer(conn, filePath)
+		fileToSync := getFileList()
+		err := sendFile(fileToSync, conn)
+		if err != nil {
+			return
+		}
 	case 3:
-		pullFromServer(conn)
+		headerMap := readHeader(conn)
+		handleRequest(headerMap, conn)
```

Comment:
Ik zou hier een default instellen welke de melding geeft dat de keuze niet goed is.

Reply:
Heb ze toegevoegd, echter word de error al eerder ondervangen. Maar tis een kleine toevoeging.